src/explore-education-statistics-frontend/src/modules/data-catalogue/components/__tests__/ReleaseStep.test.tsx

```diff
@@ -0,0 +1,201 @@
+import ReleaseStep from '@frontend/modules/data-catalogue/components/ReleaseStep';
+import { InjectedWizardProps } from '@common/modules/table-tool/components/Wizard';
+import { Release } from '@common/services/publicationService';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import noop from 'lodash/noop';
+
+describe('ReleaseStep', () => {
+  const testReleases: Release[] = [
+    {
+      id: 'test-rel-3',
+      published: '2021-01-01T11:21:17.7585345',
+      slug: 'test-rel-3-slug',
+      title: 'Another Release',
+    } as Release,
+    {
+      id: 'test-rel-1',
+      latestRelease: true,
+      published: '2021-06-30T11:21:17.7585345',
+      slug: 'test-rel-1-slug',
+      title: 'Release 1',
+    } as Release,
+    {
+      id: 'test-rel-2',
+      published: '2021-05-30T11:21:17.7585345',
+      slug: 'test-rel-2-slug',
+      title: 'Release 2',
+    } as Release,
+  ];
+
+  const wizardProps: InjectedWizardProps = {
+    shouldScroll: true,
+    stepNumber: 1,
+    currentStep: 1,
+    setCurrentStep: noop,
+    isActive: true,
+    isLoading: false,
+    goToNextStep: noop,
+    goToPreviousStep: noop,
+  };
+
+  test('renders with releases', () => {
+    render(
+      <ReleaseStep releases={testReleases} onSubmit={noop} {...wizardProps} />,
```

Comment:
Would suggest spread wizard props first so that they don't accidentally override other properties that we have explicitly set - same for the other test cases.